app/controllers/application_controller.rb

```diff
@@ -430,8 +430,12 @@ def all_identifiers
     ].filter { |e| e && !e.to_s.empty? }.uniq
   end
 
-  def after_sign_in_path_for(_user)
-    @after_login_path || hub_assigned_clients_path
+  def after_sign_in_path_for(user)
+    if !user.admin? && user.forced_password_reset_at.nil?
```

Comment:
The text of the story actually says we should only reset passwords for people who currently have weak passwords; we should change this line to exercise the password integrity validation so we only redirected to the reset if the password was weak

Reply:
Good point - we'll be looking into doing this!